.gitignore

```diff
@@ -129,3 +129,6 @@ dmypy.json
 .pyre/
 
 .idea/
+
+# pre-commit
+.pre-commit-config.yaml
```

Comment:
in open-atmos projects, we store this file in the repo, which allows to hint developers which versions to use to reflect the team's setup

Reply:
done